Tests/CredentialsTests/DummyTokenPlugin.swift

```diff
@@ -47,9 +47,9 @@ public class DummyTokenPlugin : CredentialsPluginProtocol {
                 let userProfile = UserProfile(id: "123", displayName: "Dummy User", provider: self.name)
                 let newCacheElement = BaseCacheElement(profile: userProfile)
                 #if os(OSX)
-                    self.usersCache!.setObject(newCacheElement, forKey: token as NSString)
+                    self.usersCache?.setObject(newCacheElement, forKey: token as NSString)
```

Comment:
I believe the answer is "yes". User profiles can typically be retrieved using access token against the auth provider's API, independent of whether or not a user cache is available in the application.